infra/database/mongo-database-service/MongoDatabaseService.ts

```diff
@@ -0,0 +1,125 @@
+import mongoose, { Mongoose } from 'mongoose';
```

Comment:
I believe this is the biggest change, @philarmour. Another significant update involves create-stripe-participants-sessions.ts, in addition to cleaning up the webhook. The rest are just routine tasks.